dask/dataframe/io/tests/test_parquet.py

```diff
@@ -2358,3 +2358,23 @@ def test_filter_nonpartition_columns(
     df_read = ddf_read.compute()
     assert len(df_read) == len(df_read[df_read["time"] < 5])
     assert df_read["time"].max() < 5
+
+
+@pytest.mark.parametrize("dtype", ["Int64", "str"])
```

Comment:
Can avoid the parametrization by doing both in the same dataframe

```python
df = pd.DataFrame({"A": pd.array([1, 2]), "B": pd.array(["a", "b"])})
...
```

Reply:
Makes sense -  I was trying to isolate the two different cases, but I agree your suggestion is much cleaner :)